store/wallet.ts

```diff
@@ -220,13 +220,97 @@ export default class Wallet extends VuexModule {
     return connection;
   }
 
+  @Action
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
This function indeed used `this`, why do we need this comment?

Reply:
[🔥 Remove unused eslint-disable comments](https://github.com/likecoin/app-like-co/pull/474/commits/3c503f279abf7736b110d71552bdd0c5ac72018e)